include/beast/http/detail/rfc7230.hpp

```diff
@@ -318,20 +319,23 @@ increment()
     {
         ++it;
         if(it == end)
-            return err();
+            break;
         if(! detail::is_tchar(*it))
             break;
     }
     auto const p1 = it;
+    v.first = { &*p0, static_cast<std::size_t>(p1 - p0) };
     detail::skip_ows(it, end);
     if(it == end)
-        return err();
+        return;
```

Comment:
You should get used to `void` returns, they are already possible in the language and soon they will be treatable as first class types, see http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0146r1.html